src/NVMLIOGroup.cpp

```diff
@@ -0,0 +1,673 @@
+                                  string_format_double
+                                  }},
+                              {"NVML::UTILIZATION_ACCELERATOR", {
+                                  "Percentage of time the accelerator operated on a kernel in the last set of driver samples",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::POWER", {
+                                  "Accelerator power usage in watts",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::sum,
+                                  string_format_double
+                                  }},
+                              {"NVML::POWER_LIMIT", {
+                                  "Accelerator power limit in watts",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::sum,
+                                  string_format_double
+                                  }},
+                              {"NVML::FREQUENCY_MEMORY", {
+                                  "Accelerator memory frequency in hertz",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::THROTTLE_REASONS", {
+                                  "Accelerator clock throttling reasons",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::TEMPERATURE", {
+                                  "Accelerator temperature in degrees Celsius",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::average,
+                                  string_format_double
+                                  }},
+                              {"NVML::TOTAL_ENERGY_CONSUMPTION", {
+                                  "Accelerator energy consumption in joules since the driver was loaded",
+                                  {},
+                                  GEOPM_DOMAIN_BOARD_ACCELERATOR,
+                                  Agg::sum,
+                                  string_format_double
+                                  }},
+                              {"NVML::PERFORMANCE_STATE", {
+                                  "Accelerator performance state",
```

Comment:
What is a performance state, and should it be averaged?  A bit more descriptions would be nice, units?

Reply:
Changed to expect_same, description updated